be/src/main/java/yeonba/be/exception/ExceptionAdvice.java

```diff
@@ -0,0 +1,177 @@
+package yeonba.be.exception;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.ConstraintViolationException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.ServletWebRequest;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+import yeonba.be.util.CustomResponse;
+
+@RestControllerAdvice(annotations = {RestController.class})
+public class ExceptionAdvice extends ResponseEntityExceptionHandler {
+
+  @ExceptionHandler(value = GeneralException.class)
+  public ResponseEntity<Object> handleGeneralException(
+      GeneralException exception,
+      HttpServletRequest request
+  ) {
```

Comment:
다음 PR부터 수정해서 메서드 선언할 때 라인 컨벤션 맞춰주세요. 현재 PR 다른 부분도 마찬가지 입니다.
```suggestion
      HttpServletRequest request) {
```